client/components/AuthBar/AuthBar.jsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
+import './AuthBar.scss';
+import { Link } from 'react-router-dom';
+import React from 'react';
+
+function AuthBar(props) {
+  const { username, onLogout } = props;
+
+  if (username) {
+    return (
+      <div className="auth-bar">
+        <p className="auth-bar__username">{username}</p>
+        <button type="button" className="auth-button auth-button_logout" onClick={onLogout}>
+          LogOut
+        </button>
+      </div>
+    );
+  }
+
+  return (
+    <div className="auth-bar">
```

Comment:
@IgorChaikin Could you add `Link` to `/admin` section which is visible only for `admin` users